integration/cri-containerd/test.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# 
+# Copyright (c) 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x
+set -eE
+
+err_report() {
+	cat /tmp/test-cri/cri-containerd.log
```

Comment:
Looking at `hack/test-cri.sh`, this is the default, but we can change that by setting `REPORT_DIR`. I think we should do that as this directory may already exist for example. Alternatively, I'd delete `/tmp/test-cri/` before running the test.